integration_tests/expr_05.py

```diff
@@ -69,6 +71,11 @@ def main0():
     assert a == 0
     b **= 4
     assert b == 256
+    # Test Issue 1562
+    assert ((-8)%3) == 1
+    assert ((8)%-3) == -1
+    assert abs((11.0%-3.0) - (-1.0)) < eps
+    assert abs((-11.0%3.0) - (1.0)) < eps
```

Comment:
A case for both less than 0 makes sense?

Reply:
Yes, I added 1 such test.